docs/glossary.md

```diff
@@ -295,7 +295,7 @@ _Tags: [term](/tags/term), [explanation](/tags/explanation)_
 
 #### [Persistence](/clusters#persistence)
 
-The Temporal Persistence store is a database used by Temporal Services to persist events generated and processed in the Temporal Cluster and SDK.
+The Temporal Persistence store is a database used by Temporal Services to persist events generated and processed in the Temporal Service and SDK.
 
 _Tags: [term](/tags/term), [explanation](/tags/explanation)_
```

Comment:
I think we can remove this from the glossary altogether - "Persistence" is not a core term for us.